iceoryx2/src/service/ipc.rs

```diff
@@ -48,17 +48,16 @@ pub struct Service {
 }
 
 impl crate::service::Service for Service {
-    type StaticStorage = static_storage::file::Storage;
-    type ConfigSerializer = serialize::toml::Toml;
-    type DynamicStorage = dynamic_storage::posix_shared_memory::Storage<DynamicConfig>;
-    type ServiceNameHasher = hash::sha1::Sha1;
-    type SharedMemory = shared_memory::posix::Memory<PoolAllocator>;
-    type ResizableSharedMemory =
-        resizable_shared_memory::dynamic::DynamicMemory<PoolAllocator, Self::SharedMemory>;
-    type Connection = zero_copy_connection::posix_shared_memory::Connection;
-    type Event = event::unix_datagram_socket::EventImpl;
-    type Monitoring = monitoring::file_lock::FileLockMonitoring;
-    type Reactor = reactor::posix_select::Reactor;
+    type StaticStorage = static_storage::recommended::Ipc;
+    type ConfigSerializer = serialize::recommended::Recommended;
+    type DynamicStorage = dynamic_storage::recommended::Ipc<DynamicConfig>;
+    type ServiceNameHasher = hash::recommended::Recommended;
```

Comment:
Curious why choosing not to follow the same convention as others, and have this be `hash::recommended::Ipc` ?

Could have the recommended be the same for both `Ipc` and `Local`.

Reply:
Since a hash is not a concept related to inter-process communication. A static storage can be accessed from multiple processes, is identified by name etc. but a has just defines how a set of data is hashed. The same thing goes for serializer.